addons/rating/models/mail_thread.py

```diff
@@ -174,15 +173,14 @@ def rating_apply(self, rate, token=None, rating=None, feedback=None,
     def message_post(self, **kwargs):
         rating_id = kwargs.pop('rating_id', False)
         rating_value = kwargs.pop('rating_value', False)
-        rating_feedback = kwargs.pop('rating_feedback', False)
         message = super(MailThread, self).message_post(**kwargs)
 
         # create rating.rating record linked to given rating_value. Using sudo as portal users may have
         # rights to create messages and therefore ratings (security should be checked beforehand)
         if rating_value:
             self.env['rating.rating'].sudo().create({
                 'rating': float(rating_value) if rating_value is not None else False,
-                'feedback': rating_feedback,
+                'feedback': kwargs.get("body"),
```

Comment:
As feedback is a text field and input html, should probably convert it into plaintext, like
```suggestion
                'feedback': html2plaintext(kwargs.get("body") or ""),
```

Reply:
I think all the feedbacks are shown as a message finally, so the html to text part is handling via the body of the message. Do we need to consider it here too?

Reply:
Rating views in backend display "feedback" field, so we should avoid seeing "<p>Coucou, this course is great !</p>", as it is a text field content will be raw (see for example rating_rating_view_form view). So I think we have to convert it before storing it indeed. Body on message will be sanitized, but that's on message model, rating can be displayed on its own.

Reply:
It's done :)